ports/qtactiveqt/vcpkg.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "qtactiveqt",
+  "version-string": "6.1.0",
```

Example:
```suggestion
  "version-semver": "6.1.0",
```
Then rerun `./vcpkg x-add-version --overwrite-version qtactiveqt`.